### ISSUE TITLE
Unable to save device configuration read from 2413S PLM in web interface.

### ISSUE THREAD

**Issue by rterbush** [kuestess/homebridge-platform-insteonlocal#290]
After reading devices from a PLM and selecting "Add all to config" from the dropdown, the web interface hangs with dialog box showing "Updating device...". Nothing is ever written to the config.json.

Running version 0.5.7

**Comment by rterbush** [comment]
I believe I have figured out that this was caused by the leak sensors that were in the config. Until I went through process to put these each in setup mode and query the device, I was unable to save the config. After going through that process for all leak sensors, the config was saved as expected.

Closing

**Comment by kuestess** [comment]
@rterbush Yes, battery powered devices like the leak sensor don't listen unless you wake them, or you happen to catch them  when they wake up to send a 'heartbeat' packet (indicating that they are still alive, along with battery info).  Glad you got it to work.

**Comment by rterbush** [comment]
This seems to be a general problem with using the UI to do any other configuration if you have these battery powered devices. As I remember, the ISY I have been using for the past decades had ability to ignore these battery powered devices somehow when saving changes. Not sure if it is possible to bring that same functionality forward.

As it is, I have about 10 of these leak detectors scattered around. Not possible to run around and put them all in wake mode to make other config changes...
